clash/bloody_grail.tex

```diff
@@ -90,9 +90,9 @@ \subsection*{\MakeUppercase{Additional Rules}}
     \end{enumerate}
     \item \textbf{[Sacrifice]} Remove a * Faction Unit Card from your Unit Deck in exchange for:
     \begin{enumerate}
-      \item *\svgunit{bronze}: Gain 6 \svg{gold}, 3 \svg{building_materials}, and 1 \svg{valuables}. Additionally, if the Unit Card was on the Pack side, \textbf{Search (4)} the Minor Artifact Deck.
-      \item *\svgunit{silver}: Gain 12 \svg{gold}, 6 \svg{building_materials}, and 2 \svg{valuables}. Additionally, if the Unit Card was on the Pack side, \textbf{Search (3)} the Major Artifact Deck.
-      \item *\svgunit{golden}: Gain 18 \svg{gold}, 9 \svg{building_materials}, and 3 \svg{valuables}. Additionally, \textbf{Search (2)} the Relic Card Deck. Finally, if the Unit Card was on the Pack side, \textbf{Search (2)} the \svgunit{azure} Unit Deck; you may Recruit one of these Units for half the cost (rounded down).
+      \item *\bronze\: Gain 6 \svg{gold}, 3 \svg{building_materials}, and 1 \svg{valuables}. Additionally, if the Unit Card was on the Pack side, \textbf{Search (4)} the Minor Artifact Deck.
+      \item *\silver\: Gain 12 \svg{gold}, 6 \svg{building_materials}, and 2 \svg{valuables}. Additionally, if the Unit Card was on the Pack side, \textbf{Search (3)} the Major Artifact Deck.
+      \item *\golden\: Gain 18 \svg{gold}, 9 \svg{building_materials}, and 3 \svg{valuables}. Additionally, \textbf{Search (2)} the Relic Card Deck. Finally, if the Unit Card was on the Pack side, \textbf{Search (2)} the \azure\ Unit Deck; you may Recruit one of these Units for half the cost (rounded down).
```

Comment:
That's probably fine to replace svgunit with separate command for each tier, but wasn't really necessary. This created quite a big diff for .po files

Reply:
That's probably true, but contributors often forget to use \svgunit and use \svg instead (different size). Part of a user experience, I guess.